frontend/components/propositions/PropositionDetail.tsx

```diff
@@ -62,11 +65,31 @@ const PropositionDetail: FC<PropositionDetailProps> = ({ proposition }: Proposit
           />
         </div>
       )}
-      <div
-        dangerouslySetInnerHTML={{ __html: proposition.elaboration }}
-        style={{ margin: "1.5rem 0" }}
-      />
+
+      <Box m={"1.5rem 0"}>
+        <ModuleHTML html={proposition.elaboration} />
+      </Box>
+
       {!!proposition.arguments?.length && <ArgumentSet argumentSet={proposition.arguments} />}
+      {proposition.postscript && <p dangerouslySetInnerHTML={{ __html: proposition.postscript }} />}
+
+      {!!proposition.cachedTags?.length && <TagList topics={proposition.cachedTags} />}
+
+      <footer className="footer sources-footer">
+        <ol className="citations">
+          {proposition.cachedCitations.map((citation) => {
+            const id = `citation-${citation.id}`;
+            return (
+              <li
+                className="source"
+                id={id}
+                key={id}
+                dangerouslySetInnerHTML={{ __html: citation.html }}
+              />
+            );
+          })}
+        </ol>
+      </footer>
```

Comment:
This stuff came from `OccurrenceDetail`, which is should basically the same component right now, but they might diverge later.